### PR TITLE
chore(ccore-1335): update spanner library and add json field type

### PR DESCRIPTION
**Pull request by rgoularti**
## Jira issue
[CCORE-1335](https://unicoidtech.atlassian.net/browse/CCORE-1335)

## Change description

Update `google-cloud-spanner` library to latest version and add support to `JSON` field type.

### Before

Error parsing JSON types.

image

### After
image

[CCORE-1335]: https://unicoidtech.atlassian.net/browse/CCORE-1335?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by vinicius-mariano — APPROVED** [review]
Uma lenda!! E já corrigiu nosso problema com o json! Brabo! 🚀🚜

File: setup.py [hunk 1]
```diff
@@ -17,18 +17,18 @@
...
+    version="0.5.3",
     description="Basic ORM for Spanner",
     maintainer="Aniruddha Maru",
     maintainer_email="aniruddhamaru@gmail.com",
     url="https://github.com/maroux/python-spanner-orm",
     packages=["spanner_orm", "spanner_orm.admin"],
     include_package_data=True,
     python_requires="~=3.7",
-    install_requires=["wheel", "google-cloud-spanner >= 1.6, <2.0.0dev"],
+    install_requires=["wheel", "google-cloud-spanner>=3.44.0,<4.0.0"],
```
Comment (heltonrlustosa) [inline setup.py:28]: Foi da 1.6 para 3.44? Nossa isso que é um up hahahah
Boa!! ❤️

**Review by heltonrlustosa — APPROVED** [review]

File: spanner_orm/field.py [hunk 2]
```diff
@@ -194,25 +194,8 @@ def ddl(size="MAX") -> str:
...
-    def grpc_type() -> type_pb2.Type:
-        return type_pb2.Type(code=type_pb2.BYTES)
-
-    @staticmethod
-    def validate_type(value) -> None:
-        if not isinstance(value, bytes):
-            raise error.ValidationError("{} is not of type bytes".format(value))
-
-
-class Bytes(FieldType):
```
Comment (jtoss) [inline spanner_orm/field.py:206]: que loucura, essa classe tava duplicada :thinking:

**Review by jtoss — APPROVED** [review]
Boa @rgoularti :medal_sports: , vale linkar esse PR com essa tarefa tbm:
https://unicoidtech.atlassian.net/browse/CCORE-1323

**Pull request merged**